package.json

```diff
@@ -39,7 +39,8 @@
 		"build": "vite build",
 		"test": "vitest",
 		"preview": "vite preview",
-		"prepare": "husky install"
+		"prepare": "husky install",
+		"host": "vite --host"
```

Comment:
curious - where did this package come from?

Reply:
@djtaylor8 I included the script for `vite --host` which exposes the host. Turns out when using Vite, this is the way to access local host on a mobile device so we can see our edits on our phone in real time.

Reply:
Didn't know this, thank you for sharing your insight on developing mobile friendly apps throughout this last week!